GrandCommonDivisor/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using GrandCommonDivisor.Core;
+using GrandCommonDivisor.Core.models;
+
+namespace GrandCommonDivisor.ConsoleUI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            var isFirstValid = false;
+            var first = new NumberParseResult();
```

Comment:
You don't need to create extra object here, leave it null

Reply:
Fixed.